File: 최민정/문제/1주차/17413/17413.py
s = list(input())

stack=''
result=''
flag=False # <가 있었는지 어떤지를 판별

for i in s:
  if flag==False:
    if i=='<': 
      flag=True
      stack=stack+i
    elif i==' ':
      stack=stack+i
      result=result+stack
      stack=''
    else:
      stack=i+stack

  elif flag==True:
    stack=stack+i
    if i=='>':
      flag=False
      result=result+stack
      stack=''

print(result+stack)

File: 이지홍/문제/2주차/1978/17413.js
const input = require("fs").readFileSync("/dev/stdin").toString().trim();
var I = input.split("\n");
var N = Number(I.shift());
I = I[0].split(" ").map(Number);
var count = 0;
for (let i = 0; i < N; i++) {
  let flag = 1;
  if (I[i] == 1) continue;
  for (let j = 2; j <= Math.floor(Math.sqrt(I[i])); j++) {
    if (I[i] % j === 0) flag = 0;
  }
  if (flag) count++;
}
console.log(count);